### ISSUE TITLE
OMG not transferring from L2 to L1

### ISSUE THREAD

**Issue by ShreddedLettuce** [omgnetwork/optimism#580]
Having an issue with my L2 Bridge to L1.

Started: Nov 11, 2021 6:07 PM
Completion after: Nov 11, 2021 6:34 PM

Step 1, Have SR Hash, Still Pending @ 9:04 PM

Transaction Hash: 0x86b25e5593d28f69ad679279d9d24ec9f5d625303bfdb2d4fa7106d582fd7aef

2064.685 OMG Transferred. Please Help.

**Comment by smoothsellin** [comment]
Same issue. 

Transaction Hash: 0xbd749492c758867ff51eaae20c9b7fbd3ca7264c31c41a7b4abd40cb7b7f12e9

Bridge to L1
Started: Nov 11, 2021 8:42 PM
Completion after: Nov 11, 2021 8:52 PM

Step 0, No SR Hash yet, Pending

Details

3588.770 OMG

Please help.

**Comment by craigmember** [comment]
same tried to bridge my OMG from layer2 back to layer1. 

Transaction hash:    0xb5aedec24d150eb4dee7ff0f32817eb30839799c3034c2fd3a2f9d1154f0fe1a

The transfer says confirmed on block 16649 but pending on the main page. Scared that I got rugged. 
0x3a92ca39476ff84dc579c868d4d7de125513b034  This is the address that the OMG got sent to.

**Comment by smoothsellin** [comment]
Why did ShreddedLettuce close this?

**Comment by craigmember** [comment]
I have no idea I need help

**Comment by craigmember** [comment]
This seems too legit to be a rug. IDK

**Comment by smoothsellin** [comment]
I opened a new issue. #581

I don;t think it's a rug but why are we the only 3 on Github dealing with this? I would assume a lot more people are on Boba network and are trying to move back to L1

**Comment by ShreddedLettuce** [comment]
Apologies for closing the whole thread. 30 minutes after creating it and reaching out here, it finally came through. I don't think anyone from the Boba Team had anything to do with it. Maybe the network is/was just that backed up after the snapshot. Not a rug though.

**Comment by ShreddedLettuce** [comment]
Also, I did the 'Fast Bridge' earlier. Might want to re-think the name on that one...

**Comment by Chambers-Chambers** [comment]
im currently dealing with the same issues 
Classic 7-day Bridge to L1
Started: Nov 11, 2021 5:33 PM
Completion after: Nov 18, 2021 5:52 PM
...its 9:36pm still hasnt landed in my wallet
